pkgs/servers/sql/postgresql/patches/hardcode-pg_config-paths.patch

```diff
@@ -0,0 +1,70 @@
+diff --git a/src/bin/pg_config/pg_config.c b/src/bin/pg_config/pg_config.c
+index 62b97af..e5b71e5 100644
+--- a/src/bin/pg_config/pg_config.c
++++ b/src/bin/pg_config/pg_config.c
+@@ -112,6 +112,11 @@ advice(void)
+ 	fprintf(stderr, _("Try \"%s --help\" for more information.\n"), progname);
+ }
+ 
++static char*
++get_config_val(ConfigData configdata) {
```

Comment:
Also why is this passed by reference here?

Reply:
It is passed by value, is not it? We could pass a pointer to it if we want to avoid copying the whole struct but I do not think this is worth it – the functions will probably only be called at extension build time and compiler might inline the function anyway.

Reply:
meh, I mixed them up, I meant pass-by-value and wanted to ask why :upside_down_face: